packages/components/src/components/button/button.tsx

```diff
@@ -198,19 +201,22 @@ export class Button {
         {this.styles && <style>{this.styles}</style>}
 
         {this.href ? (
-          <a
-            ref={(el) => (this.focusableElement = el)}
-            class={this.getCssClassMap()}
-            href={this.href}
-            download={this.download}
-            target={this.target}
-            rel={this.target === '_blank' ? 'noopener noreferrer' : undefined}
-            part={basePart}
-            tabIndex={this.innerTabindex}
-          >
-            <slot />
-          </a>
-        ) : (
+            <a
+              ref={(el) => (this.focusableElement = el)}
+              class={this.getCssClassMap()}
+              href={this.href}
```

Comment:
I would make this `href={this.disabled ? null : this.href}`, effectively the same as removing the attribute from the host without touching the host, which would be impolite…